packages/muelu/src/Misc/MueLu_InterfaceAggregationFactory_decl.hpp

```diff
@@ -56,6 +56,51 @@ namespace MueLu
 /*!
     @class InterfaceAggregationFactory class.
     @brief Factory for building aggregates for Lagrange multipliers.
+
+    ## Context, assumptions, and use cases ##
+
+    This factory is intended to be used for saddle-point systems of surface-coupled problems,
+    where constraints are enforced with Lagrange multipliers.
+    In addition to the primal unknowns, Lagrange multipliers are considered as dual unknowns.
+    The presenence of Lagrange mulitpliers make this a prima/dual problem
```

Comment:
> The presenence of Lagrange mulitpliers make this a prima/dual problem

Should be replaced by:
The presence of Lagrange multipliers make this a primal/dual problem.